lib/phut/open_vswitch.rb

```diff
@@ -47,10 +58,10 @@ def options
          --pidfile=#{pid_file}
          --verbose=ANY:file:dbg
          --verbose=ANY:console:err
-         --log-file=#{Phut.settings['LOG_DIR']}/open_vswitch.#{@dpid}.log
+         --log-file=#{Phut.settings['LOG_DIR']}/open_vswitch.#{name}.log
          --datapath-id=#{dpid_zero_filled}
-         --unixctl=#{Phut.settings['SOCKET_DIR']}/open_vswitch.#{@dpid}.ctl
-         netdev@vsw_#{@dpid} tcp:127.0.0.1:6633)
+         --unixctl=#{Phut.settings['SOCKET_DIR']}/open_vswitch.#{name}.ctl
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.